src/EditorFeatures/CSharp/AutomaticCompletion/Sessions/CurlyBraceCompletionSession.cs

```diff
@@ -224,6 +224,19 @@ public override AdjustNewLinesOperation GetAdjustNewLinesOperation(SyntaxToken p
                 return base.GetAdjustNewLinesOperation(previousToken, currentToken, optionSet, nextOperation);
             }
 
+            public override void AddAlignTokensOperations(List<AlignTokensOperation> list, SyntaxNode node, OptionSet optionSet, NextAction<AlignTokensOperation> nextOperation)
```

Comment:
In Smart indentation, we expect the code to be properly nested and indented
Before:
```C#
namespace ns
{
    public class C1
    { $$}
}
```
After Return:
```C#
namespace ns
{
    public class C1
    {
        $$
    }
}
```

In the case of the Block Indentation, we have code like
Before:
```C#
namespace ns
{
public class C1
{ $$}
}
```

Without this change the resultant code After Return will look like
```C#
namespace ns
{
public class C1
{
$$
    }
}
```

This is because the indentation finder thinks that the close brace has to be placed according to the nested indentation. This is not true

Instead we would like to see
```C#
namespace ns
{
public class C1
{
$$
}
}
```

Another Scenario is
Before:
```C#
namespace ns
{
        public class C1
        { $$}
}
```

After:
```C#
namespace ns
{
        public class C1
        {
        $$
        }
}
```

We dont want to see it in the base rules because the closing brace will align with the opening brace everytime the user types '}' individually.